rpm/wpa_supplicant-quiet-scan-results-message.patch

```diff
@@ -0,0 +1,30 @@
+From 763a4ef660e2bd81f6cdc71a2f29a0a3e71b2ebc Mon Sep 17 00:00:00 2001
+From: Dan Williams <dcbw@redhat.com>
+Date: Tue, 22 Nov 2016 15:48:17 +0100
+Subject: [PATCH 1/2] quiet an annoying and frequent syslog message
```

Comment:
Fedora commit e7c2862af3318e4

Author: Daniel Williams <dcbw@fedoraproject.org>  2009-01-30 19:45:46
    - Fix PEAP connections to Windows Server 2008 authenticators (rh #465022)
    - Stop supplicant on uninstall (rh #447843)
    - Suppress scan results message in logs (rh #466601)

And the last item https://bugzilla.redhat.com/show_bug.cgi?id=466601 
complaining about seeing  too many CTRL-EVENT-SCAN-RESULTS. I'm not seeing such now in our journal so wonder how much this patch really helps.

Reply:
This has been spamming logs on various devices. Cannot just remember which they were, and are they EOL devices. This is only the other half of the changes to stop the message spam as INFO. If we need them, enable as DEBUG. Our journal is anyways having too much content, a small change but having this in logs is not really necessary - when you are not debugging.